openpype/tools/publisher/widgets/overview_widget.py

```diff
@@ -336,13 +340,30 @@ def _on_instance_context_change(self):
         self.instance_context_changed.emit()
 
     def _on_convert_requested(self):
-        _, _, convertor_identifiers = self.get_selected_items()
-        self._controller.trigger_convertor_items(convertor_identifiers)
+        self.convert_requested.emit()
 
     def get_selected_items(self):
+        """Selected items in current view widget.
+
+        Returns:
+            tuple[list[str], bool, list[str]]: Selected items. List of
+                instance ids, context is selected, list of selected legacy
+                convertor plugins.
+        """
+
         view = self._subset_views_layout.currentWidget()
         return view.get_selected_items()
 
+    def get_selected_legacy_convertors(self):
+        """Selected legacy convertor identifiers.
```

Comment:
Still not sure, what this actually returns. Maybe some example of legacy convertor identifier?